plugins/importexport/native/filter/NativeXmlReviewAssignmentFilter.inc.php

```diff
@@ -0,0 +1,223 @@
+<?php
+
+/**
+ * @file plugins/importexport/native/filter/NativeXmlReviewAssignmentFilter.inc.php
+ *
+ * Copyright (c) 2014-2018 Simon Fraser University
+ * Copyright (c) 2000-2018 John Willinsky
+ * Distributed under the GNU GPL v2. For full terms see the file docs/COPYING.
+ *
+ * @class NativeXmlReviewAssignmentFilter
+ * @ingroup plugins_importexport_native
+ *
+ * @brief Base class that converts a Native XML document to a set of review assignments
+ */
+
+import('lib.pkp.plugins.importexport.native.filter.NativeImportFilter');
+
+class NativeXmlReviewAssignmentFilter extends NativeImportFilter {
+	/**
+	 * Constructor
+	 * @param $filterGroup FilterGroup
+	 */
+	function __construct($filterGroup) {
+		$this->setDisplayName('Native XML review assignment import');
+		parent::__construct($filterGroup);
+	}
+
+	//
+	// Implement template methods from NativeImportFilter
+	//
+	/**
+	 * Return the plural element name
+	 * @return string
+	 */
+	function getPluralElementName() {
+		return 'review assignments';
```

Comment:
I don't think this is expected where it's called to include spaces. See `lib/pkp/plugins/importexport/native/filter/NativeImportFilter.inc.php`.